src/avatar.cpp

```diff
@@ -818,6 +854,28 @@ void avatar::do_read( item &book )
                                    "A chapter of this book takes %d minutes to read.", reading->time ),
                  reading->time );
 
+        if( book.type->use_methods.find( "MA_MANUAL" ) != book.type->use_methods.end() ) {
+
+            const matype_id style_to_learn = martial_art_learned_from( *book.type );
+            std::string diff;
+            if( style_to_learn->learn_difficulty <= 2 ) {
+                diff = _( "easy" );
+            } else if( style_to_learn->learn_difficulty <= 4 ) {
+                diff = _( "moderately hard" );
+            } else if( style_to_learn->learn_difficulty <= 6 ) {
+                diff = _( "hard" );
+            } else if( style_to_learn->learn_difficulty <= 8 ) {
+                diff = _( "very hard" );
+            } else {
+                diff = _( "extremely hard" );
+            }
+
+            add_msg( m_info, _( "You can learn %s style from it." ), style_to_learn->name );
+            add_msg( m_info, _( "This fighting style is %s to learn." ), diff );
+            add_msg( m_info, _( "It'd be easier to master if you'd have skill expertise in %s." ),
```

Comment:
```suggestion
            add_msg( m_info, _( "It would be easier to master if you'd have skill expertise in %s." ),
```
Using this contraction feels like strange style in this context.